index.js

```diff
@@ -242,9 +249,9 @@ TCurl.prototype.request = function tcurlRequest(opts, delegate) {
         }
     });
 
-    client.waitForIdentified({
-        host: opts.hostname + ':' + opts.port
-    }, onIdentified);
+    var peer = subChan.peers.choosePeer();
```

Comment:
This has the benefit of ensuring that tcurls are evenly distributed across the cluster, at least the last recorded stable cluster, instead of hammering all health checks, for example, on host01-dc1-worker1.